README.md

```diff
@@ -0,0 +1,108 @@
+# student-service
+
+Простое gRPC-приложение на Go для управления данными студентов.
+
+## Структура проекта
+```text
+student-service/
+├── go.mod
+├── main.go
+├── proto/
+│   ├── student.proto
+│   ├── student.pb.go
+│   └── student_grpc.pb.go
+├── server/
+│   └── service.go
+```
+
+## Установка зависимостей
+
+```bash
+go mod tidy
+Генерация gRPC-кода
+Перед генерацией убедитесь, что установлены необходимые плагины:
+
+bash
+go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.3.0
+Добавьте ~/go/bin в PATH, если это не сделано:
+
+bash
+export PATH="$PATH:$(go env GOPATH)/bin"
+Сгенерируйте .pb.go файлы:
+
+bash
+protoc \
+  --go_out=paths=source_relative:. \
+  --go-grpc_out=paths=source_relative:. \
+  proto/student.proto
+Запуск сервера
+
+go run main.go
+
+gRPC-сервер будет слушать порт :50051.
+
+Включение рефлексии
+
+Для возможности тестирования с grpcurl необходимо подключить рефлексию в main.go:
+
+import "google.golang.org/grpc/reflection"
+
+...
+
+reflection.Register(grpcServer)
+
+Также необходимо установить зависимость:
```

Comment:
There is no point in documenting every runtime dependency in README
It could be done within the code itself, when required